src/app/pages/account/groups/group-details-row/group-details-row.component.spec.ts

```diff
@@ -89,10 +89,11 @@ describe('GroupDetailsRowComponent', () => {
   });
 
   it('should open edit group form', async () => {
+    const group = spectator.component.group;
```

Comment:
Better not to read from inputs, but to just use `dummyGroup` that was passed above.